geocity/apps/accounts/templates/registration/password_reset_email.html

```diff
@@ -1,5 +1,4 @@
-{% load i18n %}{% autoescape off %}
-{% translate "Bonjour," %}
+{% load i18n %}{% autoescape off %}{% translate "Bonjour" %},
```

Comment:
Don't you want to add full name also ?

Reply:
I didn't found where we call this template, it seems to be in django and not geocity.

If you can point me out something, I can take a look.

Reply:
Maybe take a look at https://github.com/django/django/blob/main/django/contrib/admin/templates/registration/password_reset_email.html

Reply:
Maybe (not tested) you can use `{{ user.get_username }}`